src/components/ha-camera-stream.ts

```diff
@@ -163,6 +184,26 @@ class HaCameraStream extends LitElement {
     }
   }
 
+  private async _renderHLSExoPlayer(url: string) {
+    this._resizeExoPlayerListener = () => this._resizeExoPlayer();
+    window.addEventListener('resize', this._resizeExoPlayerListener);
+    // https://github.com/typescript-eslint/typescript-eslint/issues/1642
+    // eslint-disable-next-line
+    setTimeout(this._resizeExoPlayerListener,200);
+    this._videoEl.style.visibility = "hidden";
+    this.hass!.auth.external!.fireMessage({
+      type: "play_hls",
+      payload: URLToolkit.buildAbsoluteURL(window.location.href, url, {alwaysNormalize: true})});
```

Comment:
No need for an external package, this is built-in to the browser.
```suggestion
      payload: new URL(url, window.location.href).toString();
```